lib/rocket_chat/server.rb

```diff
@@ -45,5 +45,9 @@ def login(username, password)
       )
       Session.new self, Token.new(response['data'])
     end
+
+    def request_json(path, options = {})
+      super(path, @options.merge(options))
```

Comment:
I suspect if needed, the safest would be to pull the source from rails vs adding another gem dep. But cross that bridge if we come to it.. there are already other instances of merge being used with the same issue. All good..